ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py

```diff
@@ -514,8 +555,8 @@ def test_torch_cross(
         frontend=frontend,
         test_flags=test_flags,
         fn_tree=fn_tree,
-        rtol=1e-2,
-        atol=1e-3,
+        rtol=1e-9,
```

Comment:
Would you revert these changes

Reply:
@hirwa-nshuti  sure, I will reput them. I asserted now in the actual function.
The problem is that I want to exclude non singular matrices.
What do you think about the new changes?

Reply:
@hirwa-nshuti 
I fixed the issues, my tests are failing only on checks that are not related to my code changes at all.
What can I do in this scenario?
Can we close #14622  #14591